recipe/meta.yaml

```diff
@@ -21,8 +21,6 @@ requirements:
     - pip
     - setuptools
     - wheel
```

Comment:
```suggestion
```
Poetry is used, so `setuptools` and `wheel` are not used.

Reply:
Thanks. I removed them